profiles/api/oauth2.py

```diff
@@ -136,4 +138,14 @@ def token() -> Response:
     filtered_json_data = {your_key: json_data[your_key] for your_key in
                           ['access_token', 'expires_in', 'name', 'orcid', 'token_type']}
 
+    try:
+        profile = Profile.query.filter_by(orcid=filtered_json_data['orcid']).one()
```

Comment:
does `.one()` mean that if there are 2 results, exception? (which shoudn't be possible due to constraints)

Reply:
Raises `MultipleResultsFound`.